extensions/src/AWSSDK.Extensions.Configuration.AWSSystemsManager/AWSSystemsManagerConfigurationProvider.cs

```diff
@@ -0,0 +1,113 @@
+﻿/*
+ * Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Sure, I'll change this. I copied these blurbs from the AWSSDK.Extensions.NETCore.Setup project which had 2016, so I left them there in case it was that way for a reason.

Reply:
Done